test/models/normalize_timdex_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+
+class NormalizeTimdexTest < ActiveSupport::TestCase
+  def timdex_records
+    VCR.use_cassette('popcorn timdex',
+                     allow_playback_repeats: true) do
+      raw_query = SearchTimdex.new.search('popcorn')
+      NormalizeTimdex.new.to_result(raw_query, 'popcorn')
+    end
+  end
+
+  def aspace_records
```

Comment:
Nice use of functions to prevent having to do this setup in each test. I often forget to do this 🍻

Reply:
Thanks for providing the timdex_records function that I could copy - I'm not sure I'd have caught this otherwise. 🍻